src/test/java/seedu/address/logic/parser/FindCommandParserTest.java

```diff
@@ -81,8 +81,14 @@ public void parse_validEmailPrefix_returnsFindCommand() {
 
     @Test
     public void parse_validAddressPrefix_returnsFindCommand() {
-        // TODO: Update test case with a/ prefix
-        assertParseFailure(parser, ADDRESS_DESC_AMY, "a/ search not implemented yet.");
+        FindCommand expectedAddressCommand =
+                new FindCommand(new AddressContainsKeywordsPredicate(Arrays.asList("Pasir", "Ris", "Bukit", "Timah")));
+
+        // no leading and trailing whitespaces
+        assertParseSuccess(parser, " a/Pasir Ris Bukit Timah", expectedAddressCommand);
+
+        // multiple whitespaces between keywords
+        assertParseSuccess(parser, " a/Pasir Ris             Bukit Timah", expectedAddressCommand);
```

Comment:
I think it would be better to have a test case of a single-word argument like " a/Pasir".